docs/prod-install.md

```diff
@@ -12,13 +12,13 @@ you need:
 * At least 2GB RAM and 10 GB disk space (4GB and 2 CPUs recommended for 100+ users).
 * A DNS name, an SSL certificate, and credentials for sending email.
 
-## Step 0: Subscribe
+## Step 1: Subscribe
```

Comment:
I'd be interested in the thoughts of other folks like @gnprice, but I think it's better to leave this as "Step 0".